src/main/java/com/google/cloud/tools/gradle/appengine/standard/AppEngineStandardPlugin.java

```diff
@@ -61,6 +61,7 @@
   @Override
   public void apply(Project project) {
     this.project = project;
+    project.getPluginManager().apply(WarPlugin.class);
```

Comment:
Just in case, this change is already in #388.

Reply:
Oh right, I built this off that branch, I'll rebase it away,